include/ginkgo/core/base/executor.hpp

```diff
@@ -54,6 +54,8 @@ struct hipblasContext;
 
 struct hipsparseContext;
 
+struct hipsparseMatDescr;
```

Comment:
Why is the matrix descriptor needed here ?

Reply:
I move the `hipsparseMatDescr` into hipsp_linops.hpp to make `handle_manager` work